ruby/test/fixtures/uninitialized_constant_suite/spec/dummy_spec.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+RSpec.describe NonExistingObject do
+  it "won't run" do
+    expect(1 + 1).to be == 2
+  end
+end
```

Comment:
I find this confusing 😕  (Figured it out later when reading the test)
```suggestion
RSpec.describe NonExistingObject do
  it "sample method on  uninitialized constant to test error output" do
    expect(1 + 1).to be == 2
  end
end
```